components/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
```

Comment:
Moved to partials folder inside templates and updated templates path to ensure they are not built to /dist/ and therefore not validated.